base/math.jl

```diff
@@ -904,16 +904,31 @@ julia> modf(-3.5)
 """
 modf(x) = isinf(x) ? (flipsign(zero(x), x), x) : (rem(x, one(x)), trunc(x))
 
-function modf(x::Float32)
-    temp = Ref{Float32}()
-    f = ccall((:modff, libm), Float32, (Float32, Ptr{Float32}), x, temp)
-    f, temp[]
-end
-
-function modf(x::Float64)
-    temp = Ref{Float64}()
-    f = ccall((:modf, libm), Float64, (Float64, Ptr{Float64}), x, temp)
-    f, temp[]
+function modf(x::T) where T<:IEEEFloat
```

Comment:
If it's derived from the musl source, maybe reference the copyright notice from there?

Reply:
Yes absolutely, I stupidly forgot in both this and and the `rem` PR.

Reply:
No longer relevant